renderdoc/driver/vulkan/wrappers/vk_device_funcs.cpp

```diff
@@ -4294,6 +4306,9 @@ VkResult WrappedVulkan::vkCreateDevice(VkPhysicalDevice physicalDevice,
   if(bufferAddressFeaturesEXT)
     bufferAddressFeaturesEXT->bufferDeviceAddressCaptureReplay = VK_TRUE;
 
+  if(multiviewPerViewViewportsFeatures)
+    multiviewPerViewViewportsFeatures->multiviewPerViewViewports = VK_TRUE;
```

Comment:
You're still enabling this feature even if the capture didn't use it. The same question applies as before - why are you forcibly enabling the extension, without any additional use on replay?

Reply:
I misunderstood the spec requirement, I thought that by including the struct in the pNexts you were automatically including the feature so I set the enablement. On rechecking khronos that is not the case for this extension, only the other extension I had been working with, so I'll remove this.

Reply:
I don't believe that is how any extensions work. For the extensions that include a feature struct (which is most of them, I think only a small number of extensions are automatically enabled just by the extension string and have no feature struct) the feature must be enabled for them to work.

That isn't the primary issue here though - what you are doing is checking for the extension struct being passed in from the capture, and forcibly turning the feature on. If the struct was passed in then most likely the bool was already true so this is redundant. If the struct were passed in but the bool was false then you're now setting it to true and *changing the behaviour of the capture*. I don't see any reason to do that and it could break the replay of captures - albeit in a somewhat obtuse circumstance.

Reply:
I agree with you, this should not be present, and is removed in the latest update. As mentioned I had misunderstood the spec requirement, I should not have been changing the behavior here at all.